packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -53,51 +61,68 @@ import { MenuToggle, Badge } from '@patternfly/react-core';
 
 ```
 
-### Primary
+### With icons
+
+Add recognizable icons to a menu toggle using the `icon` property.
```

Comment:
I went for something slightly different in my most recent push. Are there other common reasons to pass an image in, or do avatars cover the majority of cases? Mostly wondering if it could make sense to change the title to "With images" or if it's more useful to stick with the current title